### ISSUE TITLE
AVT 3 - Content Switcher Component: State and correct role are not announced by VoiceOver

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2488]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
VO announces does not announce the correct role for each button and does not announce when it is selected. 

## Expected Result
VO should announce each button as a tab using role="tab" and indicate when it is selected using aria-selected="true".
The Right and Left Arrow keys should move focus forward and backward through the tabs.

See [ARIA Authoring Practices 1.1 - Tabs ](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html) for additional ARIA requirements.
It is also recommended that the example include the tabpanel to ensure the ARIA roles and states, as well as focus are implemented correctly.

**Comment by dakahn** [comment]
Just running through some of these and doing some quick tests. This one looks good to close to me. Using VO on latest macOS the tabs are read as:
"First sections, selected, tab. You are currently on a tab..." 🎉

**Comment by snidersd** [comment]
@dakahn Test on macOS with VO and Chrome, and also tested on Win10/JAWS without issues. Issues found on iOS with VO #3530.
